ide/gsf.testrunner/nbproject/project.xml

```diff
@@ -69,6 +69,7 @@
                 </dependency>
             </module-dependencies>
             <public-packages>
+                <package>org.netbeans.modules.gsf.testrunner</package>
```

Comment:
IMHO - it is not a good idea to make a public API from the code that was not originally designed to be an API.